peepholelib/tests/sparse_svd_no_bias.py

```diff
@@ -0,0 +1,67 @@
+import importlib
+import torch
+import peepholelib
+from peepholelib.models.svd import c2s as c2s
+from time import time
+from numpy.random import randint as ri
+from torch.nn.modules.utils import _reverse_repeat_tuple
+from torch.nn.functional import pad
+importlib.reload(peepholelib.models.svd)
+if __name__ == '__main__':
+    use_cuda = torch.cuda.is_available()
+    cuda_index = torch.cuda.device_count() - 1
+    device = torch.device(f"cuda:{cuda_index}" if use_cuda else "cpu")
+    print(f"Using {device} device")
+
```

Comment:
given that this test is very similar to the previous one, I would add a boolean variable in line `15`, and use that as the `bias` parameter in line `36`.

So we do not have 2 scripts doing the same thing.